app/enterprise/2.1.x/deployment/hybrid-mode-setup.md

```diff
@@ -378,7 +378,9 @@ follow the instructions to:
     -e "KONG_CLUSTER_MTLS=pki" \
     -e "KONG_CLUSTER_SERVER_NAME=control-plane.kong.yourcorp.tld" \
     -e "KONG_CLUSTER_CERT=data-plane.crt" \
-    -e "KONG_CLUSTER_CERT_KEY=data-plane.crt"
+    -e "KONG_CLUSTER_CERT_KEY=data-plane.crt" \
```

Comment:
```suggestion
    -e "KONG_CLUSTER_CERT_KEY=/<path-to-file>/data-plane.crt" \
```
Missing the <path-to-file> in these three (unless that's on purpose?)

Reply:
no that's probably a good idea. i wasn't sure if the assumption with the --mount option was that the certs would live in the root of that mount path or in subdirs. thanks!